lib/internal/transport/RequestProcessor.ts

```diff
@@ -31,6 +31,14 @@ export function isEventIncoming(event: any): event is EventIncoming {
     return !!event.data;
 }
 
+export function workspaceId(event: CommandIncoming | EventIncoming): string {
+    if (isCommandIncoming(event)) {
+        return event.team.id;
+    } else if (isEventIncoming(event)) {
+        return event.extensions.team_id;
+    }
```

Comment:
`else`?

Reply:
I wondered about that. If I do that, it means that anything that you would force into the function via `as any` would end up in that branch and probably fail with `can not read team_id of undefined`.

Reply:
As it stands, the function can return nothing, which will likely screw up downstream code. It should probably explicitly return `undefined` and caller have to deal with that.